fabischolasi/fast-food-fast
# Develop

**What does this PR do?**
   - This pr has a signin page template, where a user can register.
   - It has a login page template, where a user can login.
   - It has a landing page, the first page that a user sees after loggin in.
   - It has a users profile page where he/she can be able to see the order history.
   - It has a page where the admin can see the list of orders, another page where an admin can accept or 
     decline an order.
   - It also has a page where an admin can view the meals available in the system, he can edit the meal, 
     or delete the meal.
   - Lastly it has a page where a user can view the menu of the day and order according to the menu.

**Description of Task to be completed?**
   - We need to add some javascript code to allow users add orders to the cart.
   - Make the static pages dynamic( bringing them to life using js)
   - Improving in the styling of each pages and make all buttons work.

**How should this be manually tested?**
   - Clone the develop branch from this repo.
   - Make sure you checkout to develop branch on your local so as you have access to the templates.
   - cd into the cloned directory and click the html files.
   - The files should open on your default browser(**the preferred one is chrome**)

**Any background context you want to provide?**
   - This pr has been build on pure html and css without any frameworks

**What are the relevant pivotal tracker stories?**
   - #160168695
   - #160168710
   - #160168545
   - #160168728
   - #160168770
   - #160168778
   - #160185075
   - #160168840

**Screenshots** 

![signup](https://user-images.githubusercontent.com/28791783/46001624-6a029300-c0b4-11e8-999c-aea8c30a68ea.png)
